components/tikv_util/src/sys/mod.rs

```diff
@@ -1,5 +1,45 @@
 // Copyright 2017 TiKV Project Authors. Licensed under Apache-2.0.
 
+mod cgroup;
+use self::cgroup::CGroupSys;
+
+const KB: u64 = 1024;
+
+pub struct SysQuota {
+    cgroup: CGroupSys,
+}
+
+impl SysQuota {
+    pub fn new() -> Self {
+        Self {
+            cgroup: CGroupSys::default(),
+        }
+    }
+
+    pub fn cpu_cores_quota(&self) -> usize {
+        let cpu_num = sysinfo::get_logical_cores();
+        let cgroup_quota = self.cgroup.cpu_cores_quota();
```

Comment:
The CGroups is a Linux specific feature, we should call this only for `#[cfg(target_os = "linux")]`

Reply:
done